src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -77,16 +77,42 @@ def load_mlflow_model(
     mlflow.set_tracking_uri(workspace.get_mlflow_tracking_uri())
 
     if model_id:
-        model = Model._get(workspace, id=model_id)
+        try:
+            model = Model._get(workspace, id=model_id)
+        except Exception as e:
+            raise UserConfigError("Unable to retrieve model by model id {} in workspace {}, error:\n{}".format(model_id, workspace.name, e))
         model_uri = "models:/{}/{}".format(model.name, model.version)
 
     if use_model_dependency:
-        pip_file = mlflow.pyfunc.get_model_dependencies(model_uri)
-        subprocess.check_call([sys.executable, "-m", "pip",
-                                "install", "-r", pip_file])
+        try:
+            pip_file = mlflow.pyfunc.get_model_dependencies(model_uri)
+        except Exception as e:
+            raise ValueError("Failed to get model dependency from given model {}, error:\n{}".format(model_uri, e))
+        try:
+            subprocess.check_output([sys.executable, "-m", "pip",
+                                    "install", "-r", pip_file])
+        except subprocess.CalledProcessError as e:
+            _logger.error("Installing dependency using requriments.txt from mlflow model failed: {}".format(e.output))
+            _classify_and_log_pip_install_error(e.output)
+            raise UserConfigError("Installing dependency using requirments.txt from mlflow model failed. "
+            "The behavior can be turned off with setting use_model_dependency to False in job spec. "
```

Comment:
v.nit: This behavior?

Reply:
Thanks! Let me update in the next PR
https://github.com/Azure/RAI-vNext-Preview/pull/154

to avoid requeuing the gated test